spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -161,4 +207,241 @@ public void setExecutorThreads(int executorThreads) {
 			this.executorThreads = executorThreads;
 		}
 	}
+
+	public static class Retry {
+
+		/**
+		 * Property for setting of the same name in {@link com.google.api.gax.retrying.RetrySettings}
```

Comment:
These all properties must also have a proper description and don't have any JavaDoc tags.
The description from here becomes a meta-data for auto-configuration properties which is used by IDEs or other visualization tools.